src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -96,6 +97,23 @@ static void do_unary_operation(cldnn::topology& topology,
     topology.add(cldnn_unary);
 }
 
+static void do_backprop_operation(cldnn::topology& topology,
+                                  const shared_ptr<Node>& op,
+                                  cldnn_activation_grad_func mode,
+                                  const cldnn_activation_additional_params& param = {0.f, 0.f})
+{
+    arguments_check(op, 2, 1);
+
+    const string& input = op->get_inputs()[0].get_tensor().get_name();
```

Comment:
Please don't use [0] because if the container empty it creates the element. Please use .at(0) because if no element exists it throw an exception.